x-pack/plugins/infra/public/pages/metrics/inventory_view/components/waffle/map.tsx

```diff
@@ -105,6 +105,7 @@ const WaffleMapOuterContainer = euiStyled.div<{ bottomMargin: number; staticHeig
   overflow-x: hidden;
   overflow-y: auto;
   margin-bottom: ${(props) => props.bottomMargin}px;
+  max-width: calc(100vw - 90px);
```

Comment:
what is this doing?

Reply:
When there is one item on the waffle map, the `Legend` component was being pushed out of the container. This makes sure that the container where the waffle map is in, considers the approximate width of the Legend component, preventing it to grow wider than it should.